release/preview/alpine310/meta.json

```diff
@@ -2,7 +2,7 @@
     "IsLinux" : true,
     "UseLinuxVersion": false,
     "PackageFormat": "powershell-${PS_VERSION}-linux-alpine-x64.tar.gz",
-    "osVersion": "Alpine",
+    "osVersion": "Alpine 3.10",
     "tagTemplates": [
         "#psversion#-alpine-3.10-#fulltag#",
```

Comment:
```suggestion
        "#psversion#-alpine-3.10-#fulltag#",
        #psversion#-alpine-3.10-#shorttag#
```

Reply:
`#psversion#-alpine-3.10-#shorttag#` would evaluate to `7.1.0-alpine-3.10-3.10` it's not a good tag, but I'll make one change to make the more obvious.

Reply:
ah I see, based on the format of how the tagtemplate was in windowsservercore, it would be `#psversion#-alpine-#shorttag#` instead of `#psversion#-alpine-3.10-#shorttag#` actually. That might help with the redundancy